core/src/main/kotlin/configuration.kt

```diff
@@ -83,7 +83,7 @@ interface DokkaConfiguration {
     interface PackageOptions {
         val prefix: String
         val includeNonPublic: Boolean
-        val reportUndocumented: Boolean
+        val reportUndocumented: Boolean?
```

Comment:
What's the point of having three-states flag instead of binary option?

Reply:
Package options have to override given "global" option from the PassConfiguration. So if it is "not set" (third option), then we have respect only what the PassConfiguration says.

Reply:
Make sense